test/commands/commit.spec.js

```diff
@@ -170,6 +191,9 @@ describe('commit command', () => {
           processEvents[signal]()
         })
 
+        execa.mockReturnValueOnce(Promise.resolve(stubs.gitAbsoluteDir))
+        fs.existsSync.mockReturnValueOnce(false).mockReturnValueOnce(false)
```

Comment:
Why we need to add those mocks inside this test ?

Reply:
to pass the is `cancelIfRebasing` test